ui-tests/src/test/java/org/openmrs/reference/page/PatientCaptureVitalsPage.java

```diff
@@ -21,6 +21,7 @@
     private static final By CONFIRM_BUTTON = By.xpath("//ul[@id='formBreadcrumb']/li[2]/span");
     private static final By CONFIRM_BUTTON_2 = By.id("coreapps-vitals-confirm");
     private static final By SAVE_BUTTON = By.xpath("//button[@type='submit']");
+    private static final By SAVE_CHANGES_BUTTON = By.xpath("//input[@type='button']");
```

Comment:
Hello @dkayiwa I have made some change on how I am accessing the button. Please look into it when you find time. 
Thank you.